documentation/global-sync-guide/TOC.md

```diff
@@ -1,3 +1,8 @@
+---
+title: RadiantOne Global Synchronization Guide
+description: RadiantOne Global Synchronization Guide
```

Comment:
Shouldn't the description be different than the title? I'm assuming these are going to be used for SEO, so we need at least _something_ in the description that gives a bit more context to what's contained in the doc. Doesn't have to be more than 1-2 sentences.